### ISSUE TITLE
File probe test failed

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#310]
I have seen failed test for file probe (`tests/probes/file`) in Jenkins. https://jenkins.open-scap.org/job/openscap-maint/394/

May be similar to still unresolved https://fedorahosted.org/openscap/ticket/442

**Comment by jan-cerny** [comment]
However, there is no coredump on that Jenkins slave :-(

**Comment by isimluk** [comment]
There is nothing we can do without coredump for this.

Do you have any idea why coredump was missing on the slave?

Can we improve the test harness that we make sure the coredump is created next time?

**Comment by ybznek** [comment]
I don't know why, we have only coredump 
```
2016-01-28_12h24m49
_openscap-1.0.11__build_tests_probes_selinuxboolean_core.4588
```

**Comment by jan-cerny** [comment]
I have looked at the test command in Jenkins job. We have `ulimit -c unlimited` command there. Then it means that the probe didn't segfault.

**Comment by jan-cerny** [comment]
Could it be that the cordump has been sent to abrt? Could there be some conflicting configuration (kernel configuration not allowing coredumps)?

**Comment by isimluk** [comment]
Look, if the file_probe had segfaulted, the openscap would tell us. The error message would be in tests/probes/file/test-suite.log or other log that captures stderr from openscap.

Do we have that stderr log saved somewhere?

**Comment by isimluk** [comment]
@ybznek nice to see we have coredump for selinuxboolean_probe. Do we have that issue tracked somewhere? Can we store that coredump from jenkins to some other place and return to it later? What was the backtrace?

**Comment by rsprudencio** [comment]
Fixed in #576
